pkg/codegen/testing/test/testdata/output-funcs/python-extras/tests/test_codegen.py

```diff
@@ -15,12 +15,13 @@
 
 import asyncio
 import json
+
+import pulumi.output
 import pytest
+from pulumi_mypkg import *
```

Comment:
Probably cleaner to just `import pulumi_mypkg` and use things directly under that namespace.

Reply:
Haven't change the style of this import, ide just decided to reorder them